### PR TITLE
Small sample updates

### PR DESCRIPTION
**Pull request by Ellord207**
UI fixes on the thematic display and screen space samples and lint warnings.

Screen space sample
- Screen space sample instructions needed to be updated.
- Screen space sample used html input element instead of ui-core Slider.
- Screen space sample effects and UI did not match when changing iModels.

Thematic Display
- Thematic Display Select components now disable options that are not supported in selected display mode.  (A feature I requested from the UI-Core team)

Lint Warnings
- Thematic Display sample had unused variables.
- cross-proding-sample/ViewCreator2d.ts Unbound methods
- cross-proding-sample/ViewCreator2d.ts tsdoc spacing.
- Image export sample UI Unbound method
- Read Settings sample  Unbound methods
- ReadSettingsUI missing semicolons
- Divider.tsx of the Swiping viewport sample called this.props in this.setState method.

### CODE REVIEW

File: src/frontend-samples/screen-space-effects-sample/ScreenSpaceEffectsUI.tsx [hunk 1]
```diff
@@ -134,15 +143,15 @@ export default class ScreenSpaceEffectsUI extends React.Component<UIProps, UISta
...
+        <Slider showMinMax={true} min={lensAngleMin} max={lensAngleMax} step={5} values={[lensAngleValue]} disabled={!this.state.enableLensDistortion} onUpdate={this._onUpdateLensAngle} />
         {this.createSlider("Strength", this.state.effectsConfig.lensDistortion.strength, 0, 1, 0.05, "enableLensDistortion", (config, val) => config.lensDistortion.strength = val)}
         {this.createSlider("Cylindrical Ratio", this.state.effectsConfig.lensDistortion.cylindricalRatio, 0, 1, 0.05, "enableLensDistortion", (config, val) => config.lensDistortion.cylindricalRatio = val)}
       </div>
     );
   }
 
   public render() {
-    const instructions = "Use the drop-down below to select which effect is applied to the viewport.";
+    const instructions = "Use the toggles below to select which effect is applied to the viewport.  Move the sliders to control those effects.";
```
Comment (pmconne) [inline src/frontend-samples/screen-space-effects-sample/ScreenSpaceEffectsUI.tsx:154]: /s/effect is/effects are

File: src/frontend-samples/screen-space-effects-sample/ScreenSpaceEffectsUI.tsx [hunk 2]
```diff
@@ -73,24 +73,27 @@ export default class ScreenSpaceEffectsUI extends React.Component<UIProps, UISta
...
       return;
 
+    // Was a new iModel opened?
+    const newViewport = this.state.viewport?.viewportId !== prevState.viewport?.viewportId;
+
     let changed = false;
     if (this.state.lensAngle !== prevState.lensAngle) {
       changed = true;
       viewport.turnCameraOn(Angle.createDegrees(this.state.lensAngle));
+      viewport.invalidateScene();
```
Comment (pmconne) [inline src/frontend-samples/screen-space-effects-sample/ScreenSpaceEffectsUI.tsx:92]: This should be `invalidateRenderPlan`. Remind me to fix `turnCameraOn` to do that itself.

File: src/frontend-samples/screen-space-effects-sample/ScreenSpaceEffectsUI.tsx [hunk 3]
```diff
@@ -121,6 +126,12 @@ export default class ScreenSpaceEffectsUI extends React.Component<UIProps, UISta
   }
 
   private getControls(): React.ReactNode {
+    // When the view is opened, the lens angel can be outside the normal range.  This will leave it unchanged until the user adjust it.
```
Comment (Josh-Schifter) [inline src/frontend-samples/screen-space-effects-sample/ScreenSpaceEffectsUI.tsx:129]: angel should be angle
adjust should be adjusts

File: src/frontend-samples/swiping-viewport/Divider.tsx [hunk 4]
```diff
@@ -78,7 +78,9 @@ export class DividerComponent extends React.Component<DividerComponentProps, {}>
     if (currentBounds.height !== prevProps.bounds.height
       || currentBounds.width !== prevProps.bounds.width) {
       const left = ((this.state.left - prevProps.bounds.left) / prevProps.bounds.width) * this.props.bounds.width + this.props.bounds.left;
-      this.setState({ bounds: this.props.bounds, left });
```
Comment (Josh-Schifter) [inline src/frontend-samples/swiping-viewport/Divider.tsx:81]: Copying props to state is usually wrong.

**Review by pmconne — APPROVED** [review]

**Review by Josh-Schifter — APPROVED** [review]

File: src/frontend-samples/screen-space-effects-sample/ScreenSpaceEffectsUI.tsx (hunk 1 above)
Comment (Ellord207) [reply src/frontend-samples/screen-space-effects-sample/ScreenSpaceEffectsUI.tsx]: After reading the instructions today, I regret that second sentence and have removed it.

**Pull request merged**